docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -333,6 +333,7 @@ Return low-level information on the container `id`
 			"NetworkDisabled": false,
 			"OnBuild": null,
 			"OpenStdin": false,
+            "PortSpecs": null,
```

Comment:
@thaJeztah do not ask me why this is displayed like this :smile: even `git diff` command do this but rendering is perfect